packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -293,11 +294,10 @@ module.exports.subscriptionsSubscriptionValidator = isA.object({
     .date()
     .timestamp('unix')
     .required(),
-  ended_at: isA.alternatives(
-    isA.date().timestamp('unix'),
-    isA.any().allow(null)
-  ),
-  nickname: isA.string().required(),
+  end_at: isA.alternatives(isA.date().timestamp('unix'), isA.any().allow(null)),
```

Comment:
Do we really want to allow `any` here, which IIUC just matches anything?

```
> end_at = isA.alternatives(isA.date().timestamp('unix'), isA.any().allow(null))
> end_at.validate({hello: "world"}).value 
{ hello: 'world' }
>
```

I *think* `isA.any().valid(null)` is what you want here instead; it's not any more intuitive but it seems to work in testing, allowing *only* `null` as the valid value.